tests/hooks/test_http_hook.py

```diff
@@ -131,6 +132,26 @@ def test_hook_contains_header_from_extra_field(self, m):
             self.assertDictContainsSubset(json.loads(expected_conn.extra), conn.headers)
             self.assertEqual(conn.headers.get('bareer'), 'test')
 
+    @requests_mock.mock()
+    @mock.patch('requests.Request')
+    def test_hook_with_method_in_lowercase(self, m, request_mock):
```

Comment:
Can we also add a simple test to assert that `get` is converted to `GET`, please?

Reply:
Added simple test to check `get` gets converted to `GET` in constructor